scapy/fields.py

```diff
@@ -1050,6 +1050,12 @@ def i2repr_one(self, pkt, x):
         return lhex(x)
 
 
+class ThreeBytesEnumField(EnumField, ThreeBytesField):
```

Comment:
Keep it `lldp.uts` as it not used elsewhere.

Reply:
addressed in 7578ef2

Reply:
uhm.. did you mean test or test & code?